Source/WebKit/WebProcess/WebPage/CoordinatedGraphics/CompositingCoordinator.cpp

```diff
@@ -161,6 +162,7 @@ void CompositingCoordinator::sizeDidChange(const IntSize& newSize)
 
 bool CompositingCoordinator::flushPendingLayerChanges(OptionSet<FinalizeRenderingUpdateFlags> flags)
 {
+    TraceScope traceScope(BackingStoreFlushStart, BackingStoreFlushEnd);
```

Comment:
Is this really equivalent to BackingStoreFlush? Again, I would use a more accurate name like `LayerFlushStart/End`

Reply:
It's not, since we now have our own category, let's add instead of reusing things.